salt/modules/virt.py

```diff
@@ -673,6 +634,117 @@ def init(name,
                     )
                     define_vol_xml_str(xml)
 
+            elif hypervisor in ['qemu', 'kvm']:
+
+                disk_type = args['format']
+                disk_file_name = '{0}.{1}'.format(disk_name, disk_type)
+                disk_size = args.get('size', None)
+
+                img_dir = __salt__['config.option']('virt.images')
+                log.debug('Image directory from config option `virt.images`'
+                          ' is {0}'.format(img_dir))
+                img_dest = os.path.join(
+                    img_dir,
+                    name,
+                    disk_file_name
+                )
+                log.debug('Image destination will be {0}'.format(img_dest))
+                img_dir = os.path.dirname(img_dest)
+                log.debug('Image destination directory is {0}'
+                          .format(img_dir))
+                try:
+                    os.makedirs(img_dir)
+                except OSError:
+                    pass
```

Comment:
Here is that commit: https://github.com/saltstack/salt/commit/6413c11f2946c20676bae69f63d539e8ba44ebea#diff-b651f4b5ad5aad0bb9fd22bacd93caa1R593